test/examples/ucp_hello_world.c

```diff
@@ -329,6 +332,14 @@ static int run_ucx_server(ucp_worker_h ucp_worker)
         goto err;
     }
 
+    test_str = (char*) calloc(test_string_length, sizeof(char));
```

Comment:
test_str is not released in case of error
also, can we avoid allocating test_str and generate the data directly to &msg->data?